server/settings/main.py

```diff
@@ -23,6 +44,16 @@ class ShotgridServiceSettings(BaseSettingsModel):
         description="The AYON user used in the services (the user corresponding to the `AYON_API_KEY` set in the service)",
     )
 
+class ShotgridCompatibilitySettings(BaseSettingsModel):
+    """ Settings to define relationships between Shotgrid and AYON.
+    """
+
+    custom_attributes_map: dict[str, str] = Field(
+        title="Custom Attributes Map",
+        default_factory = get_default_folder_attributes,
+        description="The Shotgird x AYON Custom attribute map, empty ones will be ignored.",
```

Comment:
```suggestion
        description="AYON Custom attribute <> Shotgird attribute mapping, empty ones will be ignored.",
```

Reply:
Since the left are Ayon attribute label and right SG this would be more clearly explained this way.

Reply:
I'm redoing it so it will be clearer from the UI and we won't depend on the description, but will update it nonetheless!